src/EditorFeatures/Core/Implementation/Workspaces/EditorTextFactoryService.cs

```diff
@@ -72,80 +78,12 @@ private SourceText CreateTextInternal(Stream stream, Encoding encoding, Cancella
             cancellationToken.ThrowIfCancellationRequested();
```

Comment:
default encoding is something it use if it can't figure out Encoding from stream. that is why it is called default encoding. I will add more comments.